servers/quarkus-server/src/test/java/org/projectnessie/server/AbstractRestWithMetrics.java

```diff
@@ -41,7 +41,6 @@ void smokeTestMetrics() {
 
     // then
     assertThat(body).contains("jvm_threads_live_threads");
-    assertThat(body).contains("jvm_memory_committed_bytes");
```

Comment:
Do we not report these metrics in native tests?

Reply:
GraalVM native images don't